### PR TITLE
fixedImageNotLoadedError

### PR DESCRIPTION
**Pull request by khanSoliheen**

### CODE REVIEW

File: src/components/common/HomeCard.jsx [hunk 1]
```diff
@@ -4,7 +4,10 @@ const HomeCard = ({ homeId, photoUrl, city, neighborhood, title, description, pr
     return (
         <article className="property-card">
             <article className="property-image-container">
-                <img src={photoUrl} alt="Property Image" />
+                <img src={photoUrl || `../../../public/images/house-image-mirror.png`} alt="Property Image" onError={(e) => {
+                    e.target.onerror = null; // prevents looping
+                    e.target.src = '../../../public/images/house-image-mirror.png';
```
Comment (DamyanBG) [inline src/components/common/HomeCard.jsx:9]: Why you just not import the default image with some alias/variable and reuse it? For example 
import defaultImage from ....

File: src/components/common/HomeCard.jsx [hunk 2]
```diff
@@ -4,7 +4,10 @@ const HomeCard = ({ homeId, photoUrl, city, neighborhood, title, description, pr
     return (
         <article className="property-card">
             <article className="property-image-container">
-                <img src={photoUrl} alt="Property Image" />
+                <img src={photoUrl || `../../../public/images/house-image-mirror.png`} alt="Property Image" onError={(e) => {
```
Comment (DamyanBG) [inline src/components/common/HomeCard.jsx:7]: photoUrl always will be truthy value,please, do not change src value

**Review by DamyanBG — CHANGES_REQUESTED** [review]

File: src/components/common/HomeCard.jsx (hunk 1 above)
Comment (khanSoliheen) [reply src/components/common/HomeCard.jsx]: @DamyanBG , it is done.

File: src/components/common/HomeCard.jsx (hunk 2 above)
Comment (khanSoliheen) [reply src/components/common/HomeCard.jsx]: @DamyanBG , Done.

File: src/components/common/HomeCard.jsx [hunk 3]
```diff
@@ -15,7 +16,7 @@ const HomeCard = ({ homeId, photoUrl, city, neighborhood, title, description, pr
             </article>
 
             <article className="property-info">
-                <h3>{title}</h3>
+                <h3>{title}</h3>sdfsdf
```
Comment (DamyanBG) [inline src/components/common/HomeCard.jsx]: Do we need that sdfsdf?

Comment (khanSoliheen) [reply src/components/common/HomeCard.jsx]: I have removed this.

**Review by DamyanBG — APPROVED** [review]
Good job!

**Comment by DamyanBG** [comment]
closing #386

**Pull request merged**